src/main/java/su/litvak/chromecast/api/v2/Status.java

```diff
@@ -31,9 +31,9 @@
     public final boolean standBy;
 
     Status(@JsonProperty("volume") Volume volume,
-           @JsonProperty("applications") List<Application> applications,
-           @JsonProperty("isActiveInput") boolean activeInput,
-           @JsonProperty("isStandBy") boolean standBy) {
+            @JsonProperty("applications") List<Application> applications,
```

Comment:
Please remove space/tab at the beginning of string. Basically, project uses spaces for indents.